packages/airnode-deployer/package.json

```diff
@@ -15,7 +15,7 @@
     "clean": "rimraf -rf *.tsbuildinfo ./dist ./.webpack ./build *.tgz",
     "cli": "ts-node bin/deployer.ts",
     "compile": "tsc --build tsconfig.json",
-    "copy:terraform": "copyfiles terraform/**/**/**/*.tf terraform/**/**/**/*.tpl  dist/",
+    "copy:terraform": "copyfiles terraform/**/**/*.tf terraform/**/**/*.tpl  dist/",
```

Comment:
Mhh, is the pattern "/**/" (x2) necessary? The ** stands for any nested directory so **/**  feels redundant.

Reply:
You would think so but it doesn't work that way for `copyfiles`. I can doublecheck but I'm pretty sure I had to go as deep as the directory structure was in order to not miss any files.

Reply:
Yes, it needs to be like this :man_shrugging: